--- a/freeze.h	2026-07-09 19:53:11
+++ b/freeze.h	2026-07-09 19:53:11
@@ -5,22 +5,7 @@
 # include <sys/stdtypes.h>
 #endif
 
-#ifndef __EMX__
-#ifndef getc
-# ifdef m88k                   /* Green Hill C library bug. */
-#  define getc(p)         (--(p)->_cnt < 0 ? __filbuf(p) : (int) *(p)->_ptr++)
-# else
-#  define getc(p)         (--(p)->_cnt < 0 ? _filbuf(p) : (int) *(p)->_ptr++)
-# endif
-#endif
-#ifndef putc
-# ifdef m88k                   /* Green Hill C library bug. */
-#  define putc(x, p)      (--(p)->_cnt < 0 ? __flsbuf((unsigned char) (x), (p)) : (int) (*(p)->_ptr++ = (unsigned char) (x)))
-# else
-#  define putc(x, p)      (--(p)->_cnt < 0 ? _flsbuf((unsigned char) (x), (p)) : (int) (*(p)->_ptr++ = (unsigned char) (x)))
-# endif
-#endif
-#endif
+/* SysV stdio-internal getc/putc overrides removed for modern libc */
 
 #if !defined(MSDOS) && defined(__MSDOS__)
 # define MSDOS
